modules/reporters/qmstr-reporter-html/share/skeleton/layouts/shortcodes/target-list.html

```diff
@@ -7,4 +7,8 @@ <h3>Targets:</h3>
     {{ with .Licenses -}}
         {{- delimit . ", " -}}
     {{- end -}}<br/>
+    Copyrights: 
```

Comment:
Remove the copyright data from the target-list.html. No need to print the copyright data for each target.